modules/update-using-custom-machine-config-pools-mcp-remove.adoc

```diff
@@ -32,11 +32,11 @@ error: 'node-role.kubernetes.io/worker' already has a value (), and --overwrite
 If the node does not have a `worker` label or a label from an updated MCP, add the label.
 ////
 
-. Remove the custom label from the node.
+. For each node in a custom MCP, remove the custom label from the node by running the following command:
```

Comment:
That's not a bad idea, but per the [documentation guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command) that I'm supposed to follow, in most cases I should have "by running the following command" at the end of the instruction. I think that helps keep the step focused on the action you're performing rather than the fact that you're doing it by running a command.